config.py

```diff
@@ -21,7 +21,7 @@
 if not os.environ.get('COBOT_PREFIX'):
     BOT_ALT_PREFIXES = ('cobot ', )
 
-BOT_ADMINS = os.environ.get('BOT_ADMINS', '').split() or ('meetmangukiya', )
+BOT_ADMINS = os.environ.get('BOT_ADMINS', '').split() or ('@corobo', )
```

Comment:
current `corobo` is actually @co-robo and the bot account shouldn't be its own admin IMO.

Reply:
This value has nothing to do with the account `@co-robo` used by a production instance.

This is what other people 's test instance has as a value for this field.

I am a bit interested in whether the `@` there causes any problems.

Reply:
The `@` there does no harm, but according to the docs, `corobo@localhost` is going to be safer